src/catalog/tests/snapshots/debug__stash_opened_trace.snap

```diff
@@ -151,20 +151,22 @@ Trace {
                         event: Some(
                             V1(
                                 AuditLogEventV1 {
-                                    id: 8,
-                                    event_type: Create,
-                                    object_type: Cluster,
+                                    id: 6,
+                                    event_type: Grant,
+                                    object_type: Database,
                                     user: None,
                                     occurred_at: Some(
                                         EpochMillis {
                                             millis: 0,
                                         },
                                     ),
                                     details: Some(
-                                        IdNameV1(
-                                            IdNameV1 {
-                                                id: "u1",
-                                                name: "default",
+                                        UpdatePrivilegeV1(
+                                            UpdatePrivilegeV1 {
+                                                object_id: "Du1",
+                                                grantee_id: "p",
+                                                grantor_id: "s1",
+                                                privileges: "U",
```

Comment:
What happened here? There's a lot more changes than I was expecting.

Reply:
Turns out the snapshot isn't sorted, so the large amount of changes comes from audit log events being ordered differently in the formatting of the snapshot. If you'd like I can fix the ordering in this PR, or in a follow up!

Reply:
No that's ok. As long as it's just ordering it's fine.